scripts/arangodb-bash-setup/js/collections_data_setup.js

```diff
@@ -0,0 +1,50 @@
+print(`\n>> Running Collections Setup in Arangosh as ${process.env.ARANGO_DB_USER}\n`);
+
+// Optionally load dummy data generated by app test
+let testDataFlag = false;
+
+if (process.env.ARANGO_DB_RUN_WITH_TEST_DATA == 'yes') {
+  testDataFlag = true;
+  print("Loading test data\n");
+}
+else {
+  testDataFlag = false;
+  print("Loading empty collections\n");
+}
+
+const testData = require('./test-data/testData.json');
+
+for (let collectionType in testData) {
+  for (let collectionName in testData[collectionType]) {
+    addCollectionAndData(collectionName, collectionType, testData[collectionType][collectionName]);
+  }
+}
+
+function addCollectionAndData(collectionName, collectionType, data) {
+  print(`Attempting to add ${collectionType} collection "${collectionName}" to ${process.env.ARANGO_DB_NAME}`);
+  let success = false;
+  if (collectionType == 'document') {
+    if (db._create(collectionName)) {
+      success = true;
+    }
+  }
+  else if (collectionType == 'edge') {
+    if (db._createEdgeCollection(collectionName)) {
+      success = true;
+    }
+  }
+
+  if (success) {
+    if (testDataFlag) {
+      print(`Attempting to add data to collection "${collectionName}"`);
+      data.forEach(document => {
+        if (db._collection(collectionName).save(document)) {
```

Comment:
Use a ternary for readability.
```js
condition ? print('condition is true') : print('message is false')
```

or else use the ternary expression inside the (...) in the call to print as I did above.